### PR TITLE
first iteration of tasks controller

### PR DESCRIPTION
**Pull request by mgould16**
created Crud methods.

def task_params and commented out, to iterate on w/ view. 

notice created on update / creation of task

### CODE REVIEW

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      render :new
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    if @task.update(task_params)
+      redirect_to @task, notice: 'Task was successfully updated.'
```
Comment (jamero102) [inline app/controllers/tasks_controller.rb]: redirect to task_path(@task)

Comment (jamero102) [inline app/controllers/tasks_controller.rb]:
```suggestion
      redirect_to task_path(@task), notice: 'Task was successfully updated.'
```

File: app/controllers/tasks_controller.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    redirect_to tasks_path
+  end
+
+  private
+
+  def find_task
+    @task = Task.find(params[:id])
+  end
+
+  #to complete when creating form
```
Comment (jamero102) [inline app/controllers/tasks_controller.rb]: don't think this should be commented out, we can edit as needed, it will break the method if commented

**Comment by jamero102** [comment]
have added a change to the update method and shouldn't comment out the params

File: app/controllers/tasks_controller.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  def show
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    @task = Task.new(task_params)
+
```
Comment (jamero102) [inline app/controllers/tasks_controller.rb:18]:
```suggestion
@task.user = current_user
```

**Comment by jamero102** [comment]
looks good now!

**Comment by killeraliens** [comment]
I don't see any issues

**Review by killeraliens — APPROVED** [review]
updates look good

**Comment by MaddieLewis** [comment]
Cool

**Pull request merged**